exercises/settings.gradle

```diff
@@ -158,3 +158,4 @@ include 'practice:wordy'
 include 'practice:yacht'
 include 'practice:zebra-puzzle'
 include 'practice:zipper'
+include 'practice:game-of-life'
```

Comment:
The list is alphabetically sorted. Could you please move this to keep the practice exercise list sorted?